### PR TITLE
chore(vscode): include license in packages.json

### PR DESCRIPTION
**Pull request by alestiago**
## Status

**READY**

## Description

As per the [documentation](https://code.visualstudio.com/api/references/extension-manifest):
> If you do have a LICENSE file in the root of your extension, the value for license should be "SEE LICENSE IN <filename>".

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [X] 📝 Documentation
- [X] 🗑️ Chore

### CODE REVIEW

File: extensions/vscode/package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "url": "https://github.com/VeryGoodOpenSource/dart_frog"
   },
   "homepage": "https://dartfrog.vgv.dev/",
+  "license": "SEE LICENSE IN LICENSE FILE",
```
Comment (renancaraujo) [inline extensions/vscode/package.json:16]: As we do have a SPDX recognized license, we can add its identifier:

```suggestion
  "license": "MIT",
```

Comment (alestiago) [reply extensions/vscode/package.json:16]: That was my initial commit, but I changed it to forward to LICENSE so if something is ever changed, the LICENSE file is the single point of truth.

Wdyt?

Comment (erickzanardo) [reply extensions/vscode/package.json:16]: @alestiago but I think we need to have a valid license name here right? In case someone (a store) reads this?

Comment (alestiago) [reply extensions/vscode/package.json:16]: I think both approaches are valid, but the documentation says the following:

> If you do have a LICENSE file in the root of your extension, the value for license should be "SEE LICENSE IN ".

**Review by erickzanardo — APPROVED** [review]

**Review by erickzanardo — APPROVED** [review]

**Pull request merged**